site/app/templates/admin/RainbowCustomization.twig

```diff
@@ -16,17 +16,94 @@
+                                type="button"
+                                title="Upload a Rainbow Grades customization.json directly to Submitty"
+                            >
+                                <i class="fas fa-upload" title="Download the file"></i>
+                                Upload Manual Customization
+                            </button>
+                            <button id="btn-download-customization" class="btn btn-default key_to_click" tabindex="0" type="button"
+                                    onclick="window.location.href='{{ manual_customization_download_url }}'">
+                                <i class="fas fa-download" title="Download the file"></i>
+                                Download Manual Customization
+                            </button>
+
+                            <button id="btn-download-gui-customization" class="btn btn-default key_to_click" tabindex="0" type="button"
+                                    onclick="window.location.href='{{ gui_customization_download_url }}'">
+                                <i class="fas fa-download" title="Download the file"></i>
+                                Download GUI Customization
+                            </button>
+
+                        </form>
+                    </div>
+                </div>
+            </div>
+
+            {% if manual_customization_exists %}
+            <div id="ask_which_customization" class="customization_item" style="display:block;">
+                {% else %}
+                <div id="ask_which_customization" class="customization_item" style="display:none;">
+                    {% endif %}
+                    <fieldset>
+                        <legend><h2>Customization</h2></legend>
+                        <input type="hidden" name="csrf_token" value="{{ csrfToken }}" />
+                        <form id="customizationForm">
+                            <div id="customization-options" style="display:block;">
+                                <p>It appears that a manual customization file has been provided. Please specify the preferred file to be utilized for rendering Rainbow Grades:</p>
+                                <div>
+                                    <input type="radio" id="manual_customization" name="customization" value="manual" {{ manual_customization_exists ? 'checked' : '' }}>
+                                    <label for="manual_customization">Manual Customization</label>
+                                </div>
+                                <div>
+                                    <input type="radio" id="gui_customization" name="customization" value="gui" {{ manual_customization_exists ? '' : 'checked' }}>
+                                    <label for="gui_customization">GUI Customization</label>
+                                </div>
+                            </div>
+                        </form>
+                    </fieldset>
+
+            </div>
```

Comment:
Are you missing a closing tag?

Reply:
I think it was confusing because of the indentation, fixed